README.md

```diff
@@ -17,7 +17,7 @@ The below lists all known patterns. They are grouped into four stages of maturit
 * [Review Committee](review-committee.md) - *A formal review committee is setup within an org to "officiate" particular inner source projects with resources, etc.*
 * [Service vs. library: It's inner source, not inner deployment](service-vs-library.md) - *Teams in a DevOps environment may be reluctant to work across team boundaries on common code bases due to ambiguity over who will be responsible for responding to service downtime. The solution is to realize that often it's
 possible to either deploy the same service in independent environments with separate escalation chains in the event of service downtime or factor a lot of shared code out into one library and collaborate on that.*
-* [Trusted Committer](project-roles/trusted-committer.md) - *Many inner-source projects will find themselves in a situation where they consistently receive feedback, features, and bug-fixes from contributors. In these situations project maintainers seek ways to recognize and reward the work of the contributor above and beyond single contributions.*
```

Comment:
Actually I don't think that you should change this link.  The original link describes the pattern of _having a trusted committer in the first place_.  You are changing this to link to point to _the trusted committers for this project_.  In this part of the README I think that you want the original link.  It is the part of the README that is just linking to all kinds of patterns in general, rather than information about how the repo/projects are managed.

Reply:
Yikes. You are of course right. The wrong link was exchanged.

Reply:
Great catch @rrrutledge !